### PR TITLE
[13.0][FIX] currency_rate_update: Migration to 13.0

### PR DESCRIPTION
**Pull request by alexey-pelykh**

### CODE REVIEW

File: currency_rate_update/models/res_currency_rate_provider.py [hunk 1]
```diff
@@ -195,7 +195,7 @@ def _process_rate(self, currency, rate):
 
         Module = self.env["ir.module.module"]
 
-        currency_rate_inverted = Module.sudo().search(
```
Comment (pedrobaeza) [inline currency_rate_update/models/res_currency_rate_provider.py]: Why? Previous code was ok

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
What has happened?

File: currency_rate_update/models/res_currency_rate_provider.py (hunk 1 above)
Comment (alexey-pelykh) [reply currency_rate_update/models/res_currency_rate_provider.py]: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#technical-method-to-migrate-a-module-from-120-to-130-branch sudo() is deprecated?

**Comment by alexey-pelykh** [comment]
@pedrobaeza migration seems to have been corrupted/incomplete at some point, and that needs fixing.

File: currency_rate_update/models/res_currency_rate_provider.py (hunk 1 above)
Comment (pedrobaeza) [reply currency_rate_update/models/res_currency_rate_provider.py]: sudo(user) is, not sudo(). Is documentation not clear enough?

Comment (alexey-pelykh) [reply currency_rate_update/models/res_currency_rate_provider.py]: Oh! Actually, I assumed that entire method was deprecated

Comment (alexey-pelykh) [reply currency_rate_update/models/res_currency_rate_provider.py]: Updated

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**